src/test/kotlin/jces1209/SlowAndMeaningful.kt

```diff
@@ -15,24 +16,39 @@ class SlowAndMeaningful private constructor(
     private val duration: Duration
 ) : BenchmarkQuality {
 
-    override fun provide(): VirtualUsersSource = AwsVus(duration, region,
+    override fun provide(configProperties: String?): VirtualUsersSource = AwsVus(duration, region,
         System.getenv("vpcId"),
-        System.getenv("subnetId")
+        System.getenv("subnetId"),
+        configProperties
     )
 
-    override fun behave(scenario: Class<out Scenario>): VirtualUserBehavior = VirtualUserBehavior.Builder(scenario)
+    override fun behave(scenario: Class<out Scenario>, configProperties: String?): VirtualUserBehavior = VirtualUserBehavior.Builder(scenario)
         .browser(browser)
         .load(
+            getVirtualUserLoad(configProperties)
+        )
+        .skipSetup(true)
+        .seed(12345L)
+        .build()
+
+    private fun getVirtualUserLoad(configProperties: String?): VirtualUserLoad {
+        return if (configProperties.isNullOrEmpty()) {
             VirtualUserLoad.Builder()
                 .virtualUsers(72)
                 .ramp(Duration.ofMinutes(1))
                 .maxOverallLoad(TemporalRate(15.0, Duration.ofSeconds(1)))
                 .flat(duration)
                 .build()
-        )
-        .skipSetup(true)
-        .seed(12345L)
-        .build()
+        } else {
+            VirtualUserLoad.Builder()
```

Comment:
Would it be a good idea to refactor the parameter values to variables so that we call `VirtualUserLoad.Builder()...` after the if else?